packages/flutter_tools/lib/src/test/drive_executable.dart

```diff
@@ -0,0 +1,10 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:test_core/src/executable.dart' as executable; // ignore: implementation_imports
```

Comment:
This feels a bit too cryptic. Would we recommend that our users do this too? So far the `flutter drive` command has been a fairly simple convenience command. Some users don't even use it but instead launch their apps and scripts by hand. It seems this will make it less obvious how to run things yourself. Am I missing something?

Reply:
Fair question, this is really more of a cleanup for the flutter repo. It doesn't effect users at all. 

A user can write a test which uses `package:test` and then run the test with dart or pub the same as normal. This is because the `package:test` entry point is ultimately the same as the `package:test_core` entry point.

However, this slight refactoring allows `flutter/flutter` to stop using package test and instead export `test_api` from our testing shims.

In the future we might could consider exporting `test_api` from `flutter_driver` to completely remove the need for package test. This would actually remove the ability to run the test script directly with dart. However, that is not what the change is about.

Reply:
I had another through - we could always make this a command line toggle (something like --flutter-repo), then we can have our cake and eat it too